### ISSUE TITLE
Modeling

### ISSUE THREAD

**Issue by rejasupotaro** [rejasupotaro/amazon-product-search#3]
## Metrics

- RMSE

## Loss

- MSE
- Binary Cross-Entropy
- Categorical Cross-Entropy

## Architecture

- Listwise
- Mixed Precision
- Concat
- Dot
- Multiply
- Layer Weight Regularizers
  - [Layer Weight Regularizers in Keras](https://www.kaggle.com/code/accountstatus/layer-weight-regularizers-in-keras/notebook)
- with/without Dropout
  - [Multi-Sample Dropout](https://www.ai-shift.co.jp/techblog/2170)
  - [Pitfalls with Dropout and BatchNorm in regression problems](https://towardsdatascience.com/pitfalls-with-dropout-and-batchnorm-in-regression-problems-39e02ce08e4d)
